src/Raven.Studio/wwwroot/App/views/database/settings/databaseSettings.html

```diff
@@ -0,0 +1,231 @@
+                <i class="icon-refresh"></i><span>Refresh</span>
+            </button>
+        </div>
+    </div>
+    <div class="flex-horizontal flex-noshrink summary-buttons">
+        <div class="flex-separator"></div>
+        <div data-bind="visible: viewMode() === 'summaryMode'">
+            <button class="btn btn-sm btn-default" data-bind="click: switchToEditMode" title="Edit the Database Settings">
+                <i class="icon-edit"></i><span>Edit Entries</span>
+            </button>
+        </div>
+        <div data-bind="visible: viewMode() === 'editMode', enable: !isSaveEnabled()">
+            <button class="btn btn-sm btn-default" data-bind="click: exitEditMode" title="View all effective values in a condensed view">
+                <i class="icon-table"></i><span>View Summary</span>
+            </button>
+        </div>
+    </div>
+    <div class="flex-horizontal flex-stretch-items absolute-fill content-margin entries-grid" data-bind="visible: viewMode() === 'editMode'">
+        <div class="flex-vertical">
+            <div class="categories-list-container flex-vertical flex-grow" style="position:relative;">
+                <virtual-grid class="resizable virtual-grid" params="controller: categoriesGridController"></virtual-grid>
+            </div>
+        </div>
+        <div class="flex-grow slidein-style">
+            <div data-bind="template: { name: 'db-settings-no-matching-entries-template' }, visible: !isAnyMatchingEntries()"></div>
+            <div class="panel" data-bind="visible: isAnyMatchingEntries">
+                <div class="row">
+                    <div class="col-md-5 text-center"><h4><strong>Configuration Key</strong></h4></div>
+                    <div class="col-md-7 text-center add-vertical-split"><h4><strong>Value</strong></h4></div>
+                </div>
+            </div>
+            <div class="scroll-stretch all-entries">
+                <!-- ko foreach: allEntries -->
+                <div data-bind="template: { name: 'db-settings-basic-template' }, visible: showEntry"></div>
+                <!-- /ko -->
+            </div>
+        </div>
+    </div>
+    <div class="flex-horizontal flex-stretch-items absolute-fill content-margin entries-grid" data-bind="visible: viewMode() === 'summaryMode'">
+        <div class="flex-grow flex-vertical slidein-style">
+            <div class="summary-list-container flex-vertical flex-grow" style="position:relative;">
+                <virtual-grid class="resizable virtual-grid" params="controller: summaryGridController"></virtual-grid>
+            </div>
+        </div>
+    </div>
+</div>
+
+<script type="text/html" id="db-settings-no-matching-entries-template">
+    <div class="panel">
+        <div class="panel-body">No matching entries.</div>
+    </div>
+</script>
+
+<script type="text/html" id="db-settings-basic-template">
+    <form class="flex-form entry-basic-template" autocomplete="off">
```

Comment:
we don't want to have form for every element

Reply:
done